packages/atomic/.storybook/manager.js

```diff
@@ -0,0 +1,7 @@
+import {addons} from '@storybook/addons';
+import theme from './theme';
+
+addons.setConfig({
+  enableShortcuts: false,
```

Comment:
Curious, what shortcuts does this disable?

Reply:
There's a shortcut system in storybook to open/close panels.

More than once I have found myself closing a panel and not knowing how to re-open it :D 

So I can't imagine this would work well for our end users.
That shortcut system is more helpful when you use Storybook as a development tool (and not for documentation purpose)